samples/NugetDeployment/README.md

```diff
@@ -0,0 +1,99 @@
+# Running samples
+
+From this directory, here are the steps to building and running the sample applications. By default, the containers are sending traces to `host.docker.internal`. If the agent is elsewhere (e.g. a Datadog Agent container), set the environment variable `DD_AGENT_HOST` in the docker run command.
```

Comment:
Great idea! I think it's worth doing this because it simplifies running the samples. Implemented in 2f0c4eb5af0219f6f7ba4f150fb516e4271eed8a . Thanks for the suggestion 😄